_posts/2019-05-15-cockpit-194.md

```diff
@@ -0,0 +1,53 @@
+---
+title: Cockpit 194
+author: lis
+date: 2019-05-15
+tags: cockpit linux
+slug: cockpit-194
+category: release
+summary: Firewall page improvements and a new switch
+comments: 'true'
+---
+
+Cockpit is the [modern Linux admin interface](https://cockpit-project.org/). We
+release regularly. Here are the release notes from version 194.
+
+### Firewall: Add services to a specific zone
+
```

Comment:
We also show the zones on the main firewall page now, right? I think this also deserves a screenshot and explanation.